### ISSUE TITLE
"TypeError: done is not a function" when using await

### ISSUE THREAD

**Issue by Spetnik** [flatiron/prompt#204]
When trying to use "await" (per the documentation at https://github.com/flatiron/prompt#getting-basic-prompt-information), subsequent code still runs before the prompt and then after the prompt(s) is/are entered the console returns "TypeError: done is not a function".

Using node v12.16.1:
```
const prompt = require('prompt');

const main = async () => {
  const { username, email } = await prompt.get(['username', 'email']);
  console.log(`Username: ${username}`);
  console.log(`Email: ${email}`);
};
main();
```

Returns:
```
prompt: username:  Username: undefined
Email: undefined
a
prompt: email:  b
readline.js:1170
            throw err;
            ^

TypeError: done is not a function
    at /path/to/project/.yarn/cache/prompt-npm-1.0.0-bf60634678-98e0cfc207.zip/node_modules/prompt/lib/prompt.js:336:32
    at /path/to/project/.yarn/cache/async-npm-0.9.2-d8cafe6cc3-78c0aad8ad.zip/node_modules/async/lib/async.js:154:25
    at assembler (/path/to/project/.yarn/cache/prompt-npm-1.0.0-bf60634678-98e0cfc207.zip/node_modules/prompt/lib/prompt.js:333:9)
    at /path/to/project/.yarn/cache/prompt-npm-1.0.0-bf60634678-98e0cfc207.zip/node_modules/prompt/lib/prompt.js:342:32
    at /path/to/project/.yarn/cache/prompt-npm-1.0.0-bf60634678-98e0cfc207.zip/node_modules/prompt/lib/prompt.js:625:5
    at Interface.onLine (/path/to/project/.yarn/cache/read-npm-1.0.7-950ec74ca8-78dd30f529.zip/node_modules/read/lib/read.js:111:5)
    at Interface.emit (events.js:311:20)
    at Interface._onLine (readline.js:322:10)
    at Interface._line (readline.js:699:8)
    at Interface._ttyWrite (readline.js:1025:14)
```

**Comment by ozican** [comment]
same here

**Comment by Stevemoretz** [comment]
Same as me

**Comment by Wizard13** [comment]
I confirm this, too. I bypassed it temporarily by using `promisify`

```
const prompt = require('prompt');
const {promisify} = require('util');
const promptGetAsync = promisify(prompt.get);
prompt.start();
async function main() {
	const {username, email} = await promptGetAsync(['username', 'email']);
	console.log(username, email);
}
main();
```

**Comment by ricardoekm** [comment]
Same issue here. It seems this project is no longer maintained, I've switched to [Enquirer](https://github.com/enquirer/enquirer).

**Comment by caub** [comment]
This is fixed in 1.1.0+

```js
const prompt = require('prompt');

async function main() {
	const {username, email} = await prompt.get(['username', 'email']);
	console.log(username, email);
}
main();
```
